### DISCUSSION TITLE
server dies and systemd does not restart

### DISCUSSION THREAD

**Discussion by latefield** [Nimdy/Dedicated_Valheim_Server_Script#108] [Q&A] [answered]
hey there!

I am running a server on DO using the script and I have been experiencing repeated crashes of the server which systemd does not seem to be able to recover from. Looking at the service config it looks like systemd should try to restart the service upon failure but this does not seem to happen. 

happy to provide additional information to help debug.

any thoughts or suggestions would be appreciated!

best,
Brian

**Comment by Nimdy** [comment]
what are the system resources...

**Reply by latefield** [reply]
2 GB Memory / 2 Intel vCPUs / 25 GB Disk / NYC

**Reply by latefield** [reply]
just saw that you seem to recommend 4GB of ram?

**Reply by latefield** [reply]
I will resize the droplet to 4Gb of Ram and see if reliability improves.

**Comment by Nimdy** [comment, marked as answer]
yeah that will do it!
